.golangci.yml

```diff
@@ -46,3 +46,6 @@ linters:
 #       - wrapperFunc
 #       - commentFormatting # https://github.com/go-critic/go-critic/issues/755
 
+service:
+  golangci-lint-version: 1.17.x # use the fixed version to not introduce new linters unexpectedly
```

Comment:
how often new linters are added? feel like we'll never be updating lint version if it will require us manually changing the version

Reply:
i can also disable the new linters for now and work through them this week